CL/cl_ext.h

```diff
@@ -2499,6 +2499,64 @@ clCreateBufferWithPropertiesINTEL_fn)(
     void *       host_ptr,
     cl_int *     errcode_ret) CL_API_SUFFIX__VERSION_1_0;
 
+/***********************************
+* cl_intel_program_scope_host_pipe *
+***********************************/
+#define cl_intel_program_scope_host_pipe 1
+#define CL_INTEL_PROGRAM_SCOPE_HOST_PIPE_EXTENSION_NAME "cl_intel_program_scope_host_pipe"
+
+/* New return values from clGetEventInfo when param_name is CL_EVENT_COMMAND_TYPE */
+#define CL_COMMAND_READ_HOST_PIPE_INTEL   0x4214
+#define CL_COMMAND_WRITE_HOST_PIPE_INTEL  0x4215
+#define CL_PROGRAM_NUM_HOST_PIPES_INTEL   0x4216
+#define CL_PROGRAM_HOST_PIPE_NAMES_INTEL  0x4217
+
+typedef CL_API_ENTRY cl_int (CL_API_CALL *clEnqueueReadHostPipeIntelFPGA_fn )(
```

Comment:
We shouldn't include `CL_API_ENTRY` on the typedef, only on the extern.  See: https://github.com/KhronosGroup/OpenCL-ICD-Loader/pull/189

```suggestion
typedef cl_int (CL_API_CALL *clEnqueueReadHostPipeIntelFPGA_fn )(
```

Reply:
Thanks Ben. I have addressed the comment!